posts.go

```diff
@@ -1202,12 +1212,6 @@ func viewCollectionPost(app *App, w http.ResponseWriter, r *http.Request) error
 	isMarkdown := strings.HasSuffix(slug, ".md")
 	isRaw := strings.HasSuffix(slug, ".txt") || isJSON || isXML || isMarkdown
 
-	if strings.Contains(r.URL.Path, ".") && !isRaw {
```

Comment:
should this still check if `isRaw`?

Reply:
The condition will always be false here, as it's now handled in `CollectionPostOrStatic()` in `handle.go`. The move to an additional func was needed to always allow access to static assets, while restricting access to posts if necessary.